src/content/reference/react-dom/flushSync.md

```diff
@@ -2,6 +2,10 @@
 title: flushSync
 ---
 
+<AddedInVersion version="16" />
+
+[//]: # (According to the linked answer from Dan Abramov, this was supposed to be released during v16, but is only officially documented in v18.0.0 --> https://github.com/facebook/react/issues/11527#issuecomment-360199710)
```

Comment:
Remove this and add as a review comment or comment it in mdx

```suggestion
{/*(According to the linked answer from Dan Abramov, this was supposed to be released during v16, but is only officially documented in v18.0.0 --> https://github.com/facebook/react/issues/11527#issuecomment-360199710)*/}
```

Reply:
Resolved in 2fbf483152fd0ca43fc4d399d319f295841d91b5